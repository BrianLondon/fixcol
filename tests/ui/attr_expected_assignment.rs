use fixed_derive::ReadFixed;

#[derive(ReadFixed)]
struct Thing {
    #[fixed(width = 5, align "right")]
    field: String,
}

pub fn main() {}
