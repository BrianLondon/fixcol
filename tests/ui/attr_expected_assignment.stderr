error: fixed-derive error: Expected assignment ('=' character).
 --> tests/ui/attr_expected_assignment.rs:5:30
  |
5 |     #[fixed(width = 5, align "right")]
  |                              ^^^^^^^
